zaas-service/src/test/java/org/zowe/apiml/acceptance/RefreshEndpointTest.java

```diff
@@ -44,33 +57,39 @@ void setUp() {
         initializeTls();
     }
 
+    private String getBasePath() {
+        return "https://" + hostname + ":" + port;
+    }
+
     @Nested
     @SuppressWarnings("squid:S2699") // sonar doesn't identify the restAssured assertions
     class GivenIllegalAccessModes {
+
         @Test
         void noClientCertificateGivesForbidden() {
-
+            //@formatter:off
             given().config(RestAssuredConfig.newConfig().sslConfig(new SSLConfig()))
-                .when()
-                .post(basePath + "/gateway/api/v1/auth/refresh") //REFRESH_URL
-                .then()
-                .statusCode(is(SC_FORBIDDEN));
+                .when().post(getBasePath() + "/gateway/api/v1/auth/refresh") //REFRESH_URL
```

Comment:
shouldn't be `zaas/api/v1/auth/refresh`?

Reply:
I am changing it in the next PR